File: TradeItIosTicketSDK2/CachedLinkedBrokerAccount.m
#import "CachedLinkedBrokerAccount.h"

@implementation CachedLinkedBrokerAccount

+ (BOOL)propertyIsOptional:(NSString *)propertyName {
    NSArray *optionalProperties = @[@"userCanDisableMargin", @"balanceLastUpdated", @"balance", @"fxBalance"];
    return [optionalProperties containsObject:propertyName];
}

@end

File: TradeItIosTicketSDK2/CachedLinkedBrokerAccount.swift
class CachedLinkedBrokerAccount: Codable {
    var accountName = ""
    var accountNumber = ""
    var accountIndex = ""
    var accountBaseCurrency = ""
    var balanceLastUpdated: Date?
    var balance: TradeItAccountOverview?
    var fxBalance: TradeItFxAccountOverview?
    var isEnabled = false
    var userCanDisableMargin = false
}
